terraform/custom_domains/environment_domains/workspace_variables/register_pen.tfvars.json

```diff
@@ -0,0 +1,36 @@
+{
+  "multiple_hosted_zones": true,
+  "enable_alerting": false,
+  "pg_actiongroup_name": "s189p01-rtt-production-ag",
+  "pg_actiongroup_rg": "s189p01-rtt-pd-rg",
+  "latency_threshold": 2000,
+  "percent_5xx_threshold": 20,
+  "hosted_zone": {
+    "register-trainee-teachers.education.gov.uk": {
+      "front_door_name": "s189p01-rtt-edu-domains-fd",
+      "resource_group_name": "s189p01-rttdomains-rg",
+      "domains": [
+        "pen"
+      ],
+      "cached_paths": [
+        "/packs/*"
+      ],
+      "environment_short": "pen",
+      "origin_hostname": "pen.test.teacherservices.cloud",
```

Comment:
i would have thought register-pen.test.teacherservices.cloud

Reply:
fixed and applied changes